### DISCUSSION TITLE
如何在osr模式下调用谷歌内核的打印预览

### DISCUSSION THREAD

**Discussion by 1139870644** [CefView/QCefView#406] [Q&A]
模式：osr
cef版本：113
现象：
在谷歌浏览器相同版本下点击打印，可以弹出打印预览并且，选择使用系统对话框打印
![image](https://github.com/CefView/QCefView/assets/133207503/21e99922-61dc-4aed-a9aa-3ff831996237)
打印出来是正常的

但是在框架下点击打印
![image](https://github.com/CefView/QCefView/assets/133207503/e2c98d91-3030-4597-ac92-38e24b826e03)
报错
![image](https://github.com/CefView/QCefView/assets/133207503/8bf8487e-10f6-453b-beeb-6d8d4fae192a)

点击继续后打印的报告有两条线是错位的，并且这个是必显，不论这两条线放哪
![image](https://github.com/CefView/QCefView/assets/133207503/a5a73ad5-9b11-464c-af8d-9609c6ff8143)

请问大家知道这个该怎么处理吗

**Comment by L-Super** [comment]
OSR下未验证，可以尝试：
```
//main.cpp
config.addCommandLineSwitch("--enable-print-preview");
```
